src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.UnitTests
+{
+    public class PrometheusDataModelTests
+    {
+        [Theory()]
+        [InlineData("System.Runtime", "cpu-usage", "%", "systemruntime_cpu_usage_ratio")]
+        [InlineData("System.Runtime", "working-set", "MB", "systemruntime_working_set_bytes")]
+        [InlineData("System.Runtime", "gc-heap-size", "MB", "systemruntime_gc_heap_size_bytes")]
+        [InlineData("System.Runtime", "gen-0-gc-count", "count", "systemruntime_gen_0_gc_count")]
+        [InlineData("System.Runtime", "gen-1-gc-count", "count", "systemruntime_gen_1_gc_count")]
+        [InlineData("System.Runtime", "gen-2-gc-count", "count", "systemruntime_gen_2_gc_count")]
+        [InlineData("System.Runtime", "threadpool-thread-count", "", "systemruntime_threadpool_thread_count")]
+        [InlineData("System.Runtime", "monitor-lock-contention-count", "count", "systemruntime_monitor_lock_contention_count")]
+        [InlineData("System.Runtime", "threadpool-queue-length", "", "systemruntime_threadpool_queue_length")]
+        [InlineData("System.Runtime", "threadpool-completed-items-count", "count", "systemruntime_threadpool_completed_items_count")]
+        [InlineData("System.Runtime", "alloc-rate", "B", "systemruntime_alloc_rate_bytes")]
+        [InlineData("System.Runtime", "active-timer-count", "", "systemruntime_active_timer_count")]
```

Comment:
I see some count values have a count unit and others are empty. While this is functionally the same, are you using the original metric to decide this or some other criteria?

Reply:
Yep, I got the list of default metrics from dotnet 6 web API